pkg/server/upload.go

```diff
@@ -243,25 +243,13 @@ func validateKeys(ctx context.Context, w http.ResponseWriter, keys []*pb.Tempora
 	max := ints[len(ints)-1]
 	maxEnd := max + 144
 
-	if maxEnd-min > (144 * 14) {
+	if maxEnd-min > (144 * 15) {
```

Comment:
Stick a comment on this one? I’m not understanding the switch to 15 here.

Reply:
Added: Changed from 14 to 15 because you can have a case where you submit for the past 14 days plus part of today